docs/gradle-plugin/configuring.md

```diff
@@ -93,6 +93,36 @@ koverReport {
             }
         }
 
+        // configure IC report
```

Comment:
Here and below:
it is extremely confusing to see 'IC' letters without knowing what they mean. So in comments you have to use 'Intellij Coverage binary report' everywhere. However, it seems kinda of internal feature for me. Who needs this binary report anyway, except some JB tools? Does it have publicly accessible definition/specification somewhere? If it does not, I suggest implementing the feature without advertising it — regular KDoc should be enough for internal-oriented feature.

Reply:
The reports generated here can be used in `kover-cli` to generate human-readable reports.

In general, I agree, it may be worth renaming the block to `binary {`

Reply:
I also agree that this description can be removed from here, because this is a very rare type of report that is needed in rare cases (which can be described in more detail later with the development of `kover-cli`)